File: go/fibonacci/fibonacci.go
package main

import "os"

func main() {
	n := 100000000

	a := 0
	b := 0
	c := 1

	for i := 1; i < n; i++ {
		a = b
		b = c
		c = a + b
	}

	os.Exit(c)
}

File: c/fibonacci/fibonacci.c
#include <stdlib.h>

int main() {
	int n = 100000000;

	int a = 0;
	int b = 0;
	int c = 1;

	for(int i = 1; i < n; i++) {
		a = b;
		b = c;
		c = a + b;
	}

	exit(c);
}
